ProteinCartography/api_utils.py

```diff
@@ -0,0 +1,58 @@
+from requests import Session
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+from bioservices import UniProt
```

Comment:
Ideally, I think we should move away from depending on `bioservices` fully in the future and implement our own direct API call, which I think should decrease the number of dependencies.

Reply:
(not urgent, but maybe we make an issue for it)

Reply:
Added a place holder issue: https://github.com/Arcadia-Science/ProteinCartography/issues/65